pino.js

```diff
@@ -258,6 +259,7 @@ function pino (opts, stream) {
   // internal options
   iopts.stringify = iopts.safe ? stringifySafe : JSON.stringify
   iopts.formatOpts = {lowres: true}
+  iopts.messageKeyString = ',' + JSON.stringify('' + iopts.messageKey) + ':'
```

Comment:
Are you anticipating someone using a key that would break simply doing: `iopts.messageKeyString = `,"${iopts.messageKey}":`?